src/firefly/js/ui/DropDownMenu.jsx

```diff
@@ -106,25 +102,34 @@ export class DropDownSubMenu extends PureComponent {
         return (
             <div className='ff-MenuItem ff-MenuItem-light' title={tip}>
                 <div className='menuItemText subMenu' onMouseEnter={this.show} onMouseLeave={this.hide}>
-                    <div>{text}</div>
-                    <div style={{position: 'relative', marginLeft: 10}}>
-                        <div className='arrow-right'/>
+
```

Comment:
get `this.context.dropdownDirection` in you render function not the constructor, you should use `lodash` `get`
```js
const dropdownDirection = get(this.context, 'dropdownDirection', DEFAULT_DROPDOWN_DIR)
```